src/jobservice/job/impl/legacy/legacy.go

```diff
@@ -15,7 +15,6 @@
 package legacy
 
 import "github.com/goharbor/harbor/src/pkg/scheduler"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380